pelican/contents.py

```diff
@@ -161,6 +162,21 @@ def check_properties(self):
             if not hasattr(self, prop):
                 raise NameError(prop)
 
+    def check_save_as(self):
```

Comment:
Minor nitpick but method name (`check_save_as`) and the return value (`True/False`) isn't that consistent. It would make more sense with something like `valid_save_as`. e.g. consider the later use:

```
if not content.check_save_as():
```

vs

```
if not content.valid_save_as():
```

The latter looks more meaningful semantically.